packages/frontend-v2/package.json

```diff
@@ -9,7 +9,11 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/react": "^1.8.8",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
```

Comment:
is there a reason why we are using `@emotion/styled` instead of `styled-components`? Or do they come with installing chakra-ui? Cuz if that's the case, we probably should use `@emotion/styled` since they are coupled and prob does the same thing

Reply:
Upon reading docs charkra-ui asks us to use `@emotion/styled`, and there is no documentation of whether it supports `styled-components`. Someone on github said that chakra-ui won't be supporting `styled-components`, but idk how reliable that is

Reply:
@emotion/styled is a peer dependency for chakra, in 99% of cases we should just be able to style things the Chakra way - which is use Chakra components and style through props. This should make it so that everything is consistent with our themes

Reply:
that being said if for some reason we do want some custom styling, we'll use @emotion/styled since it's already installed(as a peer dependency), we can update the notion once this is merged 👀

Reply:
oki sounds good! I guess we will just go with `chakra` + `@emotion/styled` combo!

Reply:
why does chakra require @emotion/styled and @emotion/react? since this will introduce a lot of different ways to style our components, what's the one we should be using for our codebase?

Reply:
honestly no idea but they are simple peer dependencies for chakra. the idea is to only use Chakra and not anything else

Reply:
cool, sounds good. we could also take advantage of framer later on for animations, potentially